vscode-lean4/src/utils/logger.ts

```diff
@@ -0,0 +1,18 @@
+import { Console } from 'console'
+
+class Logger extends Console {
+    constructor(stdout: NodeJS.WritableStream, stderr?: NodeJS.WritableStream) {
+        super(stdout, stderr);
+    }
+
+    log(msg: string) {
+        super.log(new Date().toLocaleTimeString(), '-', msg);
+    }
+
+    error(msg: string) {
+        super.error(new Date().toLocaleTimeString(), '-', msg);
```

Comment:
```suggestion
        super.error(new Date().toLocaleTimeString('en-US'), '-', msg);
```

Reply:
I don't really like that because it's too long, I don't really need absolute date/time anyway, this is mostly just for relative timing information - the number of seconds ticking over mostly.  Do you want the full UTC string for some other reason?
```
Wed, 13 Jul 2022 23:33:20 GMT - ...
```
I don't feel super strongly about it, just curious mostly...

Reply:
Nope, I thought it might just be a bit confusing since times will be out of whack if, say, one action runs in `us-east` and another in `us-west`. If you only need relative timestamps, many languages provide a [monotonic system clock](https://doc.rust-lang.org/std/time/struct.Instant.html). But I'm not sure JS does, so let's just go with whatever you prefer here.

Reply:
Maybe you could use `Math.round(performance.now() / 1000)` ([`performance.now`](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now))